server/src/amyserver_tools/train_mlp.py

```
@@ -983,287 +963,168 @@ def extract_landmarks_from_still(still_path: Path) -> Optional[dict]:
+def create_sliding_windows(
+    frame_vectors: List[np.ndarray], 
+    label: str, 
+    context: dict
+) -> List[Sample]:
+    """
+    Convert a sequence of normalized frame vectors into sliding window training samples.
+    
+    Strategy:
+    - Padding: Edge replication (repeat last frame) for clips shorter than WINDOW_SIZE
+    - Stride: 1 frame (generates overlapping windows for data augmentation)
+    - Output: Each Sample.landmarks contains a flattened (WINDOW_SIZE × 1629) vector
```

Ruff (0.14.10)

Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)